src/app/shared/element-child-data-classes-child-elements-list/flattened-data-classes-list.component.html

```diff
@@ -0,0 +1,339 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div class="heading-container" *ngIf="bulkActionsVisible > 0">
+  <div
+    fxFlex
+    fxLayout="row"
+    fxLayout.md="row"
+    fxLayout.sm="row"
+    fxLayout.xs="column"
+    fxLayoutAlign="space-around"
+  >
+    <div
+      fxFlex
+      fxLayout="row"
+      fxFlex="100"
+      fxFlex.md="100"
+      fxFlex.sm="100"
+      fxFlex.xs="100"
+      fxLayoutAlign="flex-start center"
+      fxLayoutAlign.xs="flex-start center"
+    >
+      <div *ngIf="isEditable && !parentDataModel.finalised" class="mt-1">
+        <button
+          mat-stroked-button
+          color="primary"
+          type="button"
+          [matMenuTriggerFor]="contentBulkActions"
+          *ngIf="bulkActionsVisible > 0"
+        >
+          Bulk actions
+          <span class="fas fa-caret-down fa-xs" style="margin-left: 2px"></span>
+        </button>
+        <mat-menu #contentBulkActions="matMenu" class="mdm--mat-menu--actions">
+          <button mat-menu-item class="warning" (click)="bulkDelete()">
+            <span class="far fa-trash-alt"></span> Delete selected rows
+          </button>
+        </mat-menu>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="table-responsive">
+  <table
+    mat-table
+    #table
+    [dataSource]="records"
+    cdkDropList
+    (cdkDropListDropped)="dropTable($event)"
+    class="mdm--mat-table mat-elevation-z3 table-striped"
+  >
+    <ng-container matColumnDef="checkbox">
+      <th
+        mat-header-cell
+        *matHeaderCellDef
+        style="width: 5%; text-align: center"
+        scope="col"
+      >
+        <mat-checkbox
+          (change)="onChecked()"
+          [(ngModel)]="checkAllCheckbox"
+          aria-label="Select all Checkboxes"
+        >
+          <span class="mdm-hidden-checkbox-label">Select all Checkboxes</span>
+        </mat-checkbox>
+      </th>
+      <td
+        mat-cell
+        *matCellDef="let record"
+        style="width: 5%; text-align: center"
+      >
+        <mat-checkbox
+          [(ngModel)]="record.checked"
+          aria-label="Select this Checkbox"
+          (change)="listChecked()"
+        >
+          <span class="mdm-hidden-checkbox-label">Select this Checkbox</span>
+        </mat-checkbox>
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="name">
+      <th
+        mat-header-cell
+        *matHeaderCellDef
+        style="max-width: 35%; width: 35%"
+        columnName="name"
+        scope="col"
+      >
+        Name
+      </th>
+      <td
+        mat-cell
+        *matCellDef="let record"
+        style="overflow-wrap: break-word; word-break: break-word"
+      >
+        <div>
+          <mdm-element-link
+            [element]="record"
+            [parentDataModel]="{ id: record.dataModel }"
+          ></mdm-element-link>
+          <div *ngIf="record.imported">
```

Comment:
I'm not sure what this `imported` state is for, probably copied from another component. This should probably be removed.